src/plugins/query_enhancements/common/types.ts

```diff
@@ -70,5 +70,8 @@ export enum S3_FIELD_TYPES {
   BINARY = 'binary',
   STRUCT = 'struct',
   ARRAY = 'array',
+  TINYINT = 'tinyint',
```

Comment:
super nit: i think it would make sense to group the similar types together. since the switch case has double tiny int smallint bigint then i would be good to group put this stuff under double as well here